contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: What is the difference between web analytics and product analytics?
+sidebar: Docs
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+[Web analytics](/web-analytics) is a streamlined experience for tracking and monitoring your most important website metrics. [Product analytics](/product-analytics) is a more powerful and flexible experience for tracking your product.
+
+When people talk about web analytics, they're generally referring to tracking visitors on your marketing website (e.g. `posthog.com` sessions). They care about things like page views, bounce rate, and top sources of traffic.
+
+Product analytics is the place to find insights about your application based on usage (e.g. `app.posthog.com` user behavior).
+
+<ComparisonTable column1="Web analytics" column2="Product analytics">
+  <ComparisonRow column1={"One, pre-built"} column2={"Unlimited, customizable"} feature="Dashboards" />
+  <ComparisonRow column1={false} column2={true} feature="Custom insights" />
+  <ComparisonRow column1={"Pageview, pageleave, conversions"} column2={"Any, custom"} feature="Events" />
+  <ComparisonRow column1={"Limited, pre-defined"} column2={"Any, custom"} feature="Properties" />
+  <ComparisonRow column1={"Small teams, marketers"} column2={"Engineering, product, data teams"} feature="Built for" />
+  <ComparisonRow column1={"At-a-glance view of website traffic"} column2={"Analysis of product usage and user behavior"} feature="Use case" />
+</ComparisonTable>
+
+## When should you use web analytics?
+
+Web analytics gives you a single, pre-built [dashboard](https://us.posthog.com/web) for a high-level overview of your site. It's about collecting data in the aggregate, as opposed to tracking specific interactions from specific users.
```

Comment:
```suggestion
Web analytics gives you a single, pre-built [dashboard](https://us.posthog.com/web) for a high-level overview of your site. It's about collecting data in the aggregate, as opposed to tracking specific interactions from specific users.

It can help you figure out which pages and products are most popular and which marketing pages resonate the most with people
```